stable/airflow/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 description: Airflow is a platform to programmatically author, schedule and monitor workflows
 name: airflow
-version: 0.17.4
+version: 0.17.5
```

Comment:
since we break worker.pod.mapping, can we bump to 0.18.0?

Reply:
Yes, according to semver I should have stepped this to 0.18.0 anyway, as this change adds a new feature.